docs/k8s.rst

```diff
@@ -116,6 +116,21 @@ All non-persisting data will be deleted, and then re-created.
 Common tasks
 ------------
 
+### Using `tutor k8s apply` with Dry Run
```

Comment:
nit: this is not rendering as a heading. You might want to try underlining it with `=`

Reply:
Nice catch.